packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,63 +1,151 @@
+| Tooltip with card appearance and general tooltip    | [EverGreen](https://evergreen.segment.com/components/tooltip)                         | This are examples within the tooltip component. This library doesn't use a button, instead just an icon. |
+| Input and guidance to add info button               | [Salesforce](https://www.lightningdesignsystem.com/components/input/)                 | This component lives within Input and shows how to build an info button instead.                         |
+| TextField with example on how to add an info button | [v8](https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield)          | This is just an example and on how to implement it.                                                      |
+| ContextualHelp                                      | [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html) | It's a full component and uses a Popover instead of tooltip.                                             |
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+#### Comparison with v0 and v8
+
+- v0 does not have an InfoButton component.
+- v8 does not have an InfoButton component, but does have a TextField component that has an example of how to add an info button to a TextField, see Open UI section above for more details.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+<InfoButton content="This is some additional information." />
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+Visual (only the button has a variant)
+
+- `transparent` (default)
+
+![transparent](./etc/images/transparent-infobutton.png)
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+#### Props
+
+```ts
+export type InfoButtonSlots = {
+  root: NonNullable<Slot<'button'>>;
+
+  /**
+   * The Popover element that wraps the content and root. Use this slot to pass props to the Popover.
+   */
+  popover: NonNullable<Slot<PopoverProps>>;
+
+  /**
+   * The content to be displayed in the PopoverSurface when the button is pressed.
+   */
+  content: NonNullable<Slot<typeof PopoverSurface>>;
+};
+
+/**
+ * InfoButton Props
+ */
+export type InfoButtonProps = ComponentProps<Partial<InfoButtonSlots>>;
+
+/**
+ * State used in rendering InfoButton
+ */
+export type InfoButtonState = ComponentState<InfoButtonSlots>;
+```
 
 ## Structure
 
-- _**Public**_
-- _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
+_**Public**_
+
+```jsx
+<InfoButton
+  content={
+    <>
+      Popover above-start lorem ipsum dolor sit amet consectetur.{' '}
+      <Link href="https://react.fluentui.dev">Learn more</Link>
```

Comment:
Is focus going to be handled to allow actionable items in the popover? I don't recall that being addressed in the description above.

Reply:
I see it mentioned below, but it might be good to call out in the initial description above (can contain focusable content)